### PR TITLE
Improved importing of inventories

### PR DESCRIPTION
**Pull request by jmcameron**
This PR improves the importing of inventories (inventory definitions, not articles of stock):

- After importing new inventories, it refreshes the inventory registry
- If successful, it reports the number of new inventories that were created
- Improved warnings about bad import files
- Adds the ability to include a column to add a tag for all new inventories

**TESTING**
- Use bhima_test (do 'yarn build:db')
- Go to:  Inventory > Inventory Registry > [Menu] -> Import Inventories
   - Note the recently improved column definitions, including the new optional 'tag' column
   - Try a test file (try the attached test file 'import-inventories-test-good.csv', which should work)
   - Note the success message
   - Try importing the "good" test file again and note that no new inventories are imported
   - There are two "bad" test files.  Please try each.   The importer will be a little more helpful.  It will report if the required column headers are missing or misspelled, it will alert you to that.  If some rows do not have data in the required columns, the importer will complain about that
   - In the Inventory Registry, enable the filter and search the "Tag" column for A to see the tag that was created due to some rows in the "good" import file having values in the "tag" column.
 
[import-inventories-test-good.csv](https://github.com/IMA-WorldHealth/bhima/files/9358095/import-inventories-test-good.csv)
[import-inventories-test-bad-header.csv](https://github.com/IMA-WorldHealth/bhima/files/9358097/import-inventories-test-bad-header.csv)
[import-inventories-test-missing-data.csv](https://github.com/IMA-WorldHealth/bhima/files/9358098/import-inventories-test-missing-data.csv)
.

### CODE REVIEW

**Review by lomamech — APPROVED** [review]
image

I checked, it works really well

File: client/src/i18n/fr/inventory.json [hunk 1]
```diff
@@ -31,15 +31,20 @@
...
         "INVENTORY_CODE_INFO" : "inventory_code : le code souhaité (identifiant abrégé) pour l'article en stock.  Si ce code ou inventory_text correspond à un inventaire existant, il sera utilisé. Sinon, une nouvelle classe d'inventaire sera créée.",
-        "INVENTORY_CONSUMABLE_INFO" : "inventory_consumable : l'article est-il consommable ? (1=oui, 0=non, optionelle, la valeur par défaut est 1)",
+        "INVENTORY_CONSUMABLE_INFO" : "inventory_consumable : (optionel) l'article est-il consommable ? (1=oui, 0=non, la valeur par défaut est 1)",
         "INVENTORY_GROUP_INFO" : "inventory_group : Si cela ne correspond pas à un nom de groupe d'inventaire existant, un nouveau groupe d'inventaire sera créé avec ce nom",
-        "INVENTORY_IS_ASSET_INFO" : "inventory_is_asset : l'article est-il un actif ? (1=oui, 0=non, optionnel, valeur par défaut 0)",
-        "INVENTORY_MODEL_INFO" : "inventory_model : le modèle du fabricant pour ce type d'article (optionnel)",
+        "INVENTORY_IMPORT_BAD_HEADERS": "ERROR : L'importation des inventaires a échoué !  Vérifiez les noms d'en-tête de colonne manquants ou mal orthographiés.",
```
Comment (lomamech) [inline client/src/i18n/fr/inventory.json]: You only forgot to translate ERROR by ERREUR

File: client/src/i18n/fr/inventory.json [hunk 2]
```diff
@@ -31,15 +31,20 @@
...
+        "INVENTORY_IMPORT_ERROR":  "ERROR : L'importation des inventaires a échoué !  Vérifiez les lignes qui ne contiennent pas de données pour l'une des colonnes requises.",
```
Comment (lomamech) [inline client/src/i18n/fr/inventory.json]: ERROR by ERREUR

**Review by lomamech — APPROVED** [review]

**Comment by lomamech** [comment]
@jmcameron I think I'm done with the review, once the change is made, the branch will be merged

File: client/src/i18n/fr/inventory.json (hunk 2 above)
Comment (jmcameron) [reply client/src/i18n/fr/inventory.json]: Good catch.  Fixed.

**Pull request merged**